pkg/cmd/util/labels.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+)
+
+// ParseLabel parses a label spec into a Set
+func ParseLabel(labelSpec string) (labels.Set, error) {
+	args := strings.Split(labelSpec, "=")
+
+	switch len(args) {
+	case 2:
+		return map[string]string{args[0]: args[1]}, nil
+	default:
+	}
+
+	return nil, fmt.Errorf("invalid label spec: %s", labelSpec)
+}
+
+var (
+	mut     sync.Mutex
+	version int
+)
+
+func GetLabel() string {
```

Comment:
it's not just new-app right?  "osc create -f myconfiglist.json" also needs labels applied.